lib/clog.rb

```diff
@@ -14,11 +14,11 @@ def self.emit(message)
         metadata.reduce({}) do |hash, item|
           case item
           when Hash
-            hash.merge(item)
+            deep_merge(hash, item)
           when Sequel::Model
-            hash.merge(serialize_model(item))
+            deep_merge(hash, serialize_model(item))
           else
-            hash.merge({invalid_type: item.class.to_s})
+            deep_merge(hash, {invalid_type: item.class.to_s})
```

Comment:
What's the use case? I know it may be the most strict interpretation of "only one interesting sub-field" as intended for clog, but I think model dumps being a `well_known_metadata_or_representation_name` per f2b128f972ce81847956e386d626bd9a94c0106e, so it'd be fine to have `{model: {...standarddump...}, going_concern: {extra}`

Reply:
`{model: {...standarddump...}, going_concern: {extra}` aligns better with Clog's philosophy. The benefits are insufficient to justify its implementation. Complicating the logging code is unwarranted, could potentially lead to more bugs. I dropped that commit.